provider/vdpa_utils.py

```diff
@@ -0,0 +1,135 @@
+from avocado.utils import process
+
+
+def check_ovs_status():
+    """
+    Check if ovs-vsctl and openvswitch service are installed and running.
+    Returns True if both are available and running, otherwise False.
+    """
+    cmd = "which ovs-vsctl && systemctl status openvswitch.service"
+    return process.system(cmd, ignore_status=True, shell=True) == 0
+
+
+def get_ovs_bridges():
+    """
+    Get all ovs bridges.
+    Returns a list of ovs bridge names.
+    """
+    cmd = "ovs-vsctl list-br"
+    return process.system_output(cmd, shell=True).decode().split()
+
+
+def get_vf_pci_address(nic_netdst):
+    """
+    Get vf pci address from a given network destination.
+    Returns the vf pci address as a string.
+    """
```

Comment:
```suggestion
    """
    Get vf pci address from a given network destination.
    Returns the vf pci address as a string.
    
    :param nic_netdst: xxxxx
    :type nic_netdst: xxxxx
    
    :return: xxx
    :rype: xxxx
    """
```

For docstring format, here's a reference for you: https://github.com/avocado-framework/avocado/blob/master/avocado/utils/process.py#L1035~#L1072

Reply:
@nickzhq I have formatted all in the file, help help review. thanks a lot .